src/fastman/commands/misc.py

```
@@ -0,0 +1,689 @@
+@register
+class PkgListCommand(Command):
+    signature = "pkg:list"
+    description = "List installed packages"
+
+    def handle(self):
+        manager, _ = PackageManager.detect()
+
+        Output.info(f"Using package manager: {manager}")
+
+        if manager == "uv":
+            subprocess.run(["uv", "pip", "list"])
```

Ruff (0.14.14)

[error] Starting a process with a partial executable path

(S607)